### PR TITLE
Java: Added Zpopmin command. (Sorted Set Commands)

### PR DESCRIPTION
**Pull request by SanHalacogluImproving**

### CODE REVIEW

File: java/client/src/main/java/glide/api/commands/SortedSetBaseCommands.java [hunk 1]
```diff
@@ -183,4 +183,42 @@ CompletableFuture<Double> zaddIncr(
...
+     */
+    CompletableFuture<Map<String, Double>> zpopmin(String key, long count);
+
+    /**
+     * Removes and returns the member with the lowest score from the sorted set stored at the
+     * specified <code>key</code>.
+     *
+     * @see <a href="https://redis.io/commands/zpopmin/">redis.io</a> for more details.
+     * @param key The key of the sorted set.
+     * @return A map containing the removed member and their corresponding score.<br>
```
Comment (acarbonetto) [inline java/client/src/main/java/glide/api/commands/SortedSetBaseCommands.java:214]:
```suggestion
     * @return A map containing the removed member and its corresponding score.<br>
```

File: java/client/src/main/java/glide/api/models/BaseTransaction.java [hunk 2]
```diff
@@ -1241,6 +1242,43 @@ public T zcard(@NonNull String key) {
         return getThis();
     }
 
+    /**
+     * Removes and returns up to <code>count</code> members with the lowest scores from the sorted set
```
Comment (acarbonetto) [inline java/client/src/main/java/glide/api/models/BaseTransaction.java:1246]: same as above please

File: java/integTest/src/test/java/glide/SharedCommandTests.java [hunk 3]
```diff
@@ -1011,4 +1011,23 @@ public void zcard(BaseClient client) {
...
+
+    @SneakyThrows
+    @ParameterizedTest
+    @MethodSource("getClients")
+    public void zpopmax(BaseClient client) {
+        String key = UUID.randomUUID().toString();
+        Map<String, Double> membersScores = Map.of("a", 1.0, "b", 2.0, "c", 3.0);
+        assertEquals(3, client.zadd(key, membersScores).get());
+        assertEquals(Map.of("a", 1.0), client.zpopmin(key).get());
+        assertEquals(Map.of("b", 2.0, "c", 3.0), client.zpopmin(key, 3).get());
```
Comment (acarbonetto) [inline java/integTest/src/test/java/glide/SharedCommandTests.java:1023]:
```suggestion
        assertEquals(Map.of("b", 2.0, "c", 3.0), client.zpopmin(key, 10).get());
```

**Review by acarbonetto — APPROVED** [review]
looks good.  Minor comments.

File: java/integTest/src/test/java/glide/SharedCommandTests.java (hunk 3 above)
Comment (SanHalacogluImproving) [reply java/integTest/src/test/java/glide/SharedCommandTests.java:1023]: I don't get what you are trying to accomplish via 10?

File: java/client/src/main/java/glide/api/models/BaseTransaction.java (hunk 2 above)
Comment (SanHalacogluImproving) [reply java/client/src/main/java/glide/api/models/BaseTransaction.java:1246]: Bit confused as to what you are referring to.

File: java/integTest/src/test/java/glide/SharedCommandTests.java (hunk 3 above)
Comment (acarbonetto) [reply java/integTest/src/test/java/glide/SharedCommandTests.java:1023]: Ignore me... 3 is good enough.

File: java/client/src/main/java/glide/api/models/BaseTransaction.java (hunk 2 above)
Comment (acarbonetto) [reply java/client/src/main/java/glide/api/models/BaseTransaction.java:1246]: Make it match: SortedSetBaseCommands.java documentation

**Pull request merged**